### DISCUSSION TITLE
Sharing data between datatables on the same page

### DISCUSSION THREAD

**Discussion by antedrnasin** [Power-Components/livewire-powergrid#1108] [Q&A]
Hi guys,

using livewire 2.x

I have 3 datatables on my page (tabs). The idea is that you select items from first table and then the second table
shows subitems of selected item, same for the last table, needs to show sub-subitems of selected subitems.

Can this be done?

**Comment by Sicklou** [comment]
Hello,

Maybe with using events and listeners on the livewire components
